lib/setup/SetupClass.php

```diff
@@ -694,6 +696,59 @@ public function drop()
         $this->removeFlatnodeFile();
     }
 
+    /**
+     * Setup settings_test.php in the build/settings directory from build/.env file
+     *
+     * @return null
+     */
+    public function setupWebsite()
+    {
+        info('Setting up website\n');
+        $fileName = CONST_InstallPath.'/.env';
+        if (!file_exists($fileName)) {
+            throw new Exception('.env File not found in build. Please copy the contents of .env.example to .env and build Nominatim again.');
+        }
+        $contents = file_get_contents($fileName);
+        $contents = explode("\n", $contents);
+
+        $op = fopen(CONST_InstallPath.'/settings/settings_test.php', 'w');
+
+        // Currently using CONST_BasePath and CONST_InstallPath.
+        // Once dotenv is setup, getenv() can be used, or another
+        // alternate option is to build settings_test.php using cmake.
+        fwrite($op, "<?php
+@define('CONST_BasePath', '".CONST_BasePath."');
+@define('CONST_InstallPath', '".CONST_InstallPath."');
+if (file_exists(getenv('NOMINATIM_SETTINGS'))) require_once(getenv('NOMINATIM_SETTINGS'));
+if (file_exists(CONST_InstallPath.'/settings/local.php')) require_once(CONST_InstallPath.'/settings/local.php');
+if (isset(\$_GET['debug']) && \$_GET['debug']) @define('CONST_Debug', true);");
+        
+
+        // Array used to store all the env variables in key-value format.
+        $envVariables = array();
+
+        foreach ($contents as $data) {
+            $arr = explode('=', $data, 2);
```

Comment:
Looks like logic to parse a `.env` file into a data structure `$envVariables`. That should move into `Nominatim::Config` so it's reusable and testable. Unless the dotenv PHP package can already parse it for us.